server/jobs/job_store.go

```diff
@@ -17,6 +18,33 @@ const (
 type Job struct {
 	Output []string
 	Status JobStatus
+
+	readIndex  int
+	readBuffer string
+}
+
+// size(p) is configured to be 5.2Mb by stow, if a line is longer than 5.2Mb
+// we hold the remaining line in the readBuffer and write it to the buffer in the
+// next iteration
+func (j *Job) Read(p []byte) (n int, err error) {
```

Comment:
Is this used in this PR?

Reply:
Yup, we pass in the job object, which implements the `Reader` interface, to `stow.Put()` method

Reply:
Why is Job implementing the reader interface? Does that make sense semantically?  Jobstore is responsibe for persisting but Job shouldn't care about the details of that which seems to be what's happening here. Let's separate this out.

Reply:
Moved the reader out of the job struct.